### PR TITLE
Golang increment to 1.14

### PR DESCRIPTION
**Pull request by duncancrawford**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ implement an algorithm with e.g. more randomness, but with longer Ids and shorte
 
 ### License and copyright
 
-	Copyright (c) 2016. Oleg Sklyar and teris.io. MIT license applies. All rights reserved.
+**[Forked][terisioshortid]:** Copyright (c) 2016. Oleg Sklyar and teris.io. MIT license applies. All rights reserved.
```
Comment (osklyar) [inline README.md:88]: Mind dropping this one?

File: go.mod [hunk 2]
```diff
@@ -0,0 +1,3 @@
+module github.com/duncancrawford/shortid
```
Comment (osklyar) [inline go.mod:1]: Could you please check what do you want to have here?

File: shortid_integration_test.go [hunk 3]
```diff
@@ -4,7 +4,7 @@
 package shortid_test
 
 import (
-	"github.com/teris-io/shortid"
+	"github.com/duncancrawford/shortid"
```
Comment (osklyar) [inline shortid_integration_test.go]: Please fix.

File: shortid_test.go [hunk 4]
```diff
@@ -4,7 +4,7 @@
 package shortid_test
 
 import (
-	"github.com/teris-io/shortid"
+	"github.com/duncancrawford/shortid"
```
Comment (osklyar) [inline shortid_test.go]: Please fix.

File: README.md [hunk 5]
```diff
@@ -95,15 +95,7 @@ inspired this "port":
 
 [go]: https://golang.org
 [nodeshortid]: https://github.com/dylang/shortid
+[terisioshortid]: https://github.com/teris-io/shortid
```
Comment (osklyar) [inline README.md:98]: I guess this was done for your fork, mind removing?

File: .github/workflows/gobuild.yml [hunk 6]
```diff
@@ -0,0 +1,99 @@
+name: build
```
Comment (osklyar) [inline .github/workflows/gobuild.yml:1]: I quite like this change, but am not convinced about the rest (with the exception of time.Since). Mind going through the PR again and checking what is worth keeping when merging from your fork?

**Review by osklyar — CHANGES_REQUESTED** [review]

**Comment by duncancrawford** [comment]
Ah it was not my intention for this to come back as a PR to you as I wasn’t sure you would want to migrate to GHA. I am happy to submit a PR so will do that shortly

File: shortid_test.go [hunk 7]
```diff
@@ -1,16 +1,15 @@
 // Copyright (c) 2016-2017. Oleg Sklyar & teris.io. All rights reserved.
 // See the LICENSE file in the project root for licensing information.
 
-package shortid_test
+package shortid
```
Comment (osklyar) [inline shortid_test.go:4]: Why is this change required? The test was only using the public API and that seems to be unchanged, so why merge the packages?

File: README.md (hunk 1 above)
Comment (osklyar) [reply README.md:88]: Is this still a part of the merge?

File: .gitignore [hunk 8]
```diff
@@ -1,3 +1,13 @@
-.idea/
-vendor/
-Gopkg.lock
+*.exe
```
Comment (osklyar) [inline .gitignore:1]: Why any of those are really required?

**Review by osklyar — CHANGES_REQUESTED** [review]
I would prefer the support for Go 1.15 to go in independently of any other changes, which are more like differences in personal preferences than objective improvements.
